Dockerfile

```diff
@@ -1,9 +1,18 @@
-FROM drujensen/crystal:0.23.1
+FROM crystallang/crystal:0.23.1
 
-WORKDIR /app/user
+# Install Dependencies
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update -qq && apt-get install -y --no-install-recommends libpq-dev libsqlite3-dev libmysqlclient-dev libreadline-dev git curl vim netcat
 
-ADD . /app/user
+# Install Node
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
+RUN apt-get install -y nodejs
```

Comment:
Why we still depend on node? Am I missing something? 😅

Reply:
We are generating webpack and use npm packages for front-end assets which is dependent on node.